versioned/storage/store/src/main/java/org/projectnessie/versioned/storage/versionstore/VersionStoreImpl.java

```diff
@@ -919,6 +921,15 @@ public ContentResult getValue(Ref ref, ContentKey key) throws ReferenceNotFoundE
     }
   }
 
+  private static ContentResult getValueNotFound(
+      ContentKey key, boolean returnNotFound, StoreIndex<CommitOp> index) {
+    if (!returnNotFound) {
+      return null;
+    }
+    IdentifiedContentKey identifiedKey = buildIdentifiedKey(key, index, null, null, x -> null);
+    return contentResult(identifiedKey, null, null);
```

Comment:
Slightly more readable:

```suggestion
    if (returnNotFound) {
      IdentifiedContentKey identifiedKey = buildIdentifiedKey(key, index, null, null, x -> null);
      return contentResult(identifiedKey, null, null);
    }
    return null;
```